x/auth/tx/service.go

```diff
@@ -49,8 +50,13 @@ const (
 
 // TxsByEvents implements the ServiceServer.TxsByEvents RPC method.
 func (s txServer) GetTxsEvent(ctx context.Context, req *txtypes.GetTxsEventRequest) (*txtypes.GetTxsEventResponse, error) {
-	offset := int(req.Pagination.Offset)
-	limit := int(req.Pagination.Limit)
+	offset := 0
```

Comment:
Update it to reuse `ParsePagination` [Comment](https://github.com/cosmos/cosmos-sdk/pull/7965#discussion_r530467064).